stix_shifter_modules/elastic_ecs/entry_point.py

```diff
@@ -11,5 +14,15 @@ def __init__(self, connection={}, configuration={}, options={}):
         if connection:
             self.setup_transmission_basic(connection, configuration)
 
+        basepath = os.path.dirname(__file__)
+        filepath = os.path.abspath(os.path.join(basepath, "stix_translation"))
+
         dialect = 'default'
-        self.setup_translation_simple(dialect_default=dialect, results_translator=ResultTranslator(options, dialect))
\ No newline at end of file
+        query_translator = QueryTranslator(options, dialect, filepath)
+        results_translator = ResultTranslator(options, dialect)
+        self.add_dialect(dialect, query_translator=query_translator, results_translator=results_translator, default_include=True, default=True)
+
+        dialect = 'beats'
+        query_translator = QueryTranslator(options, dialect, filepath)
+        results_translator = ResultTranslator(options, dialect)
+        self.add_dialect(dialect, query_translator=query_translator, results_translator=results_translator, default_include=False, default=False)
```

Comment:
if two dialects are created then you should be able to use setup_translation_simple() -

`self.setup_translation_simple(dialect_default='ecs')`

Reply:
Are you sure?  I tried using that and it wasn't working.

Is this required to merge?  I don't really have time to work on this further.

Reply:
it should work. we already have few connectors that uses the similar structure. Here are couple of examples:

https://github.com/opencybersecurityalliance/stix-shifter/blob/61e8975c39836981c61ce3f6cbd1f8bf6e2eedba/stix_shifter_modules/aws_athena/entry_point.py#L29
https://github.com/opencybersecurityalliance/stix-shifter/blob/61e8975c39836981c61ce3f6cbd1f8bf6e2eedba/stix_shifter_modules/qradar/entry_point.py#L12

This is just my suggestion to make the module structure consistent. We can do this later if you dont have time atm.